### PR TITLE
XW-251 | Show a message when user opened a section/category that does not exist

### PR DESCRIPTION
**Pull request by vforge**
https://wikia-inc.atlassian.net/browse/XW-251

### CODE REVIEW

File: front/scripts/main/routes/ApplicationRoute.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ App.ApplicationRoute = Em.Route.extend(Em.TargetActionSupport, App.TrackClickMix
 			this.controller && this.controller.hideLoader();
 
 			// Clear notification alerts for the new route
-			this.controller.get('alertNotifications').clear();
+			this.controller.get('alertNotifications').clearNotifications();
```
Comment (bognix) [inline front/scripts/main/routes/ApplicationRoute.ts]: @mari-mcmurtrie could you explain why you added this line:
```
this.controller.get('alertNotifications').clear();
```
It actually broke some functionality for us.
For instance with your implementation there is no way to add notification when some error occurs on transition.
In our case it was when user entered the page and 404 error occurred, we were showing notification on next route why transition happened.

Comment (rogatty) [inline front/scripts/main/routes/ApplicationRoute.ts]: Seems like a bad UX to hide all notifications after each transition. @mari-mcmurtrie was there a specific use case that needed this?

**Comment by rogatty** [comment]
LGTM

**Comment by bognix** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Comment by bognix** [comment]
Tests are passing locally:
```
gulp node-test
[19:20:24] Using gulpfile ~/git/wikia/mercury/gulpfile.js
[19:20:24] Starting 'node-test'...
[19:20:24] Finished 'node-test' after 3.17 ms

Summary:
┌────────────────────────────────────────────────────┬────────┬────────┬───────┬─────────┐
│ File                                               │ Failed │ Passed │ Total │ Runtime │
├────────────────────────────────────────────────────┼────────┼────────┼───────┼─────────┤
│ ...ikia/mercury/www/server/facets/auth/authView.js │ 0      │ 10     │ 10    │ 365     │
├────────────────────────────────────────────────────┼────────┼────────┼───────┼─────────┤
│ ...gna/git/wikia/mercury/www/server/lib/Caching.js │ 0      │ 11     │ 11    │ 371     │
├────────────────────────────────────────────────────┼────────┼────────┼───────┼─────────┤
│ ...bogna/git/wikia/mercury/www/server/lib/Utils.js │ 0      │ 26     │ 26    │ 392     │
├────────────────────────────────────────────────────┼────────┼────────┼───────┼─────────┤
│ ...a/git/wikia/mercury/www/server/lib/MediaWiki.js │ 0      │ 11     │ 11    │ 452     │
└────────────────────────────────────────────────────┴────────┴────────┴───────┴─────────┘
```
looks like Jenkins error

**Pull request merged**